FDBLibTLS/Makefile

```diff
@@ -1,109 +0,0 @@
-PROJECTPATH = $(dir $(realpath $(firstword $(MAKEFILE_LIST))))
```

Comment:
Should we at least move `test`, `test-plugin`, and `test-verify` to somewhere else in our build system so that we can do development on the plugin code?

Or has statically-linking the plugin meant that this testing framework no longer works?

Reply:
Given that I (and previously you) were just asked to do a few enhancements to this code, I'm more nervous than before about us deprecating testing that seems important to keep.